src/main/java/com/google/codeu/servlets/ChartServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.JsonArray;
+import com.google.codeu.utils.CsvLoader;
+
+@WebServlet("/moviechart")
+public class ChartServlet extends HttpServlet {
+
+  private JsonArray movieInfoArray;
+
+  @Override
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/movies.csv"));
```

Comment:
Make the path `"/WEB-INF/movies.csv"` a file constant var

Reply:
Also do you know if the servlet gets instantiated once when the server starts? Or instantiated for every request?

Reply:
I think whatever is in the init() function will get instantiated once the server starts?

Reply:
https://tomcat.apache.org/tomcat-5.5-doc/servletapi/javax/servlet/Servlet.html#init(javax.servlet.ServletConfig)

Seems like it is called once whenever a servlet is placed into service, which is good.

Reply:
I have made file path a static final string.